theming/theme-package.rst

```diff
@@ -462,15 +459,15 @@ As a starting point we use this rules set:
            <div class="row">
              <div class="box">
                <div class="col-xs-12 col-sm-12">
-                 <xsl:apply-templates css:select="#content"/>
+                 <xsl:apply-templates css:select="#content" />
                </div>
                <div class="clearFix"></div>
              </div>
            </div>
-           <section class="row" id="viewlet-below-content-body">
+           <section id="viewlet-below-content-body" class="row">
              <div class="box">
                <div class="col-xs-12 col-sm-12">
-                <xsl:copy-of select="//div[@id='viewlet-below-content']"/>
+                <xsl:copy-of css:select="#viewlet-below-content" />
```

Comment:
The literal xpath > css translation would but the actual transformation will happen correctly in both cases and it's simpler to understand and demonstrate in a training, do you agree?

Under the hoods it seems that the only difference would be a tiny little bit of less performance but that should be really low and if it isn't we should work on that or put real disclaimers in the docs because as far as I'm concerned the CSS selectors should resemble real CSS selector, that is, best practices should apply (and avoiding specifying the tag when an id is unique is one of them).